apps/prairielearn/src/ee/lib/validateHTML.ts

```diff
@@ -0,0 +1,602 @@
+import * as parse5 from 'parse5';
+
+type DocumentFragment = parse5.DefaultTreeAdapterMap['documentFragment'];
+type ChildNode = parse5.DefaultTreeAdapterMap['childNode'];
+
+const htmlGlobalAttributes = [
+  'accesskey',
+  'class',
+  'contenteditable',
+  'data-*',
+  'dir',
+  'draggable',
+  'enterkeyhint',
+  'hidden',
+  'id',
+  'inert',
+  'inputmode',
+  'lang',
+  'popover',
+  'spellcheck',
+  'style',
+  'tabindex',
+  'title',
+  'translate',
+];
+
+const mustacheTemplateRegex = /^\{\{.*\}\}$/;
+
+/**
+ * Checks that the required property is an int (or mustache template) or adds an error to the provided list.
```

Comment:
Please use the term "attribute", not "property", everywhere in this file.

Reply:
Changed in 8f5c2be88d306dec842f62a968bf5c24938a0cec.